cmake/leatherman.cmake

```diff
@@ -273,7 +277,11 @@ macro(gettext_compile dir inst)
                 DEPENDS ${fpath})
             add_custom_target(${lang}-${PROJECT_NAME} DEPENDS ${mo})
             add_dependencies(translations ${lang}-${PROJECT_NAME})
-            install(FILES ${mo} DESTINATION "${inst}/${lang}/LC_MESSAGES")
+            if(LEATHERMAN_LOCALE_VAR AND LEATHERMAN_LOCALE_INSTALL)
+                install(FILES ${mo} DESTINATION "${LEATHERMAN_LOCALE_VAR}/${LEATHERMAN_LOCALE_INSTALL}/${lang}/LC_MESSAGES")
```

Comment:
`LEATHERMAN_LOCALE_VAR` is expected to contain the name of an enviroment variable, that Leatherman reads at runtime to determine a starting path (with `LEATHERMAN_LOCALE_INSTALL` used to define a project-determined offset from that location).